### PR TITLE
2567 mobile first header logo language selector navigation

### PR DESCRIPTION
**Pull request by bameyrick**
### What is the context of this PR?
Updated header and navigation to be optimised for mobile, created blue ghost button, and changed beta bar to match new design.

#### Navigation
**Main nav and Page nav** - Classes have changed to allow for the addition of page nav (child pages) to be included in the mobile nav solution and general clean up to match updated designs:

- `nav--inline` replaces `nav--horizontal`.
- `nav--page` replaces `nav--sub`.
- `nav__link--current` replaces `nav__item--current` and moves to the link (page nav).
- `nav--progress` replaces `nav--vertical` (better reflects the specific use).

#### Feedback/Beta
- The inline feedback form has been removed.
- A new beta banner component has been created which is positioned above the header.

### How to review 
Check everything looks good.

### CODE REVIEW

**Comment by armstrongb** [comment]
Needs to review with internal service and sectional navigation etc.

File: components/beta-bar/beta-bar.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="beta-bar">
+  <div class="container beta-bar__inner">
+    <h3 class="beta-bar__badge">Beta</h3>
+    <p class="beta-bar__desc u-fs-s">This is a new service – your <a href="#">feedback</a> will help us improve it.</p>
+  </div>
+</div>
+
```
Comment (boxadesign) [inline components/beta-bar/beta-bar.hbs]: I think we could handle this without some of the new classes and adding a modifier to the existing `grid__col--flex` class. Have a look below:

```suggestion
<div class="beta-bar">
  <div class="container">
    <div class="grid grid--flex">
      <div class="grid__col grid__col--flex grid__col--vertical-center">
        <h3 class="beta-bar__badge">Beta</h3>
        <p class="u-fs-s u-mb-no">This is a new service – your <a href="#">feedback</a> will help us improve it.</p>
      </div>
    </div>
  </div>
</div>
```

We would need to add `grid__col--vertical-center` to `align-items: center`.

**Comment by boxadesign** [comment]
Again with the mobile nav against the title it looks unbalanced...

image

**Comment by boxadesign** [comment]
Is it the case that the mobile nav work is not in this PR?

**Comment by bameyrick** [comment]
@boxadesign We never really considered the mobile nav in this but I think we should. I'll have a chat about it with @will-madebymoon after lunch

File: components/beta-bar/beta-bar.hbs (hunk 1 above)
Comment (bameyrick) [reply components/beta-bar/beta-bar.hbs]: Thats a good suggestion... I'll push something up that utilises the flex grid more for this.

**Comment by will-madebymoon** [comment]
> Again with the mobile nav against the title it looks unbalanced...
> 
> image

We are ditching this mobile nav

**Comment by boxadesign** [comment]
@will-madebymoon So are suggesting that main navigation in services like SBR, R'Ops etc should be in the footer? I'm not sure. I'm on board with secondary links going down there but primary navigation needs to be easily accessed from the top of the page so users don't have to scroll to the bottom just to go from page to page.

image

**Comment by will-madebymoon** [comment]
> @will-madebymoon So are suggesting that main navigation in services like SBR, R'Ops etc should be in the footer? I'm not sure. I'm on board with secondary links going down there but primary navigation needs to be easily accessed from the top of the page so users don't have to scroll to the bottom just to go from page to page.
> 
> image
In terms of a user need it's probably only RH and EQ that would require any mobile optimisation.

**Comment by will-madebymoon** [comment]
> > @will-madebymoon So are suggesting that main navigation in services like SBR, R'Ops etc should be in the footer? I'm not sure. I'm on board with secondary links going down there but primary navigation needs to be easily accessed from the top of the page so users don't have to scroll to the bottom just to go from page to page.
> > image
> > In terms of a user need it's probably only RH and EQ that would require any mobile optimisation.

Though I see your point about the primary nav. Perhaps we retain the burger menu functionality for this however drop the icon and replace with the words Menu or something applicable.

**Comment by RhodGriffith** [comment]
@boxadesign Is the sign out button going in the footer for all services or will this be an option for services like EQ if there's no main nav?

**Comment by will-madebymoon** [comment]
> @boxadesign Is the sign out button going in the footer for all services or will this be an option for services like EQ if there's no main nav?

Only on mobile for EQ

**Comment by bameyrick** [comment]
@boxadesign If it's just changing the font sizes of the items within the menu I'm happy to do that.
Regarding changing the button from an icon to the word "Menu", are you happy with us to just crack on and do that @will-madebymoon on or would you like to design something first? I was thinking of just using the ghost (white outline) button?

**Comment by bameyrick** [comment]
@boxadesign Everything you've done looks fine, there are a few changes I need to make before we can merge this so will leave the do not merge label on here

**Comment by boxadesign** [comment]
@bameyrick OK. I need to add unit tests for the mobile nav also.

**Pull request merged**